src/proposal/timeline.rs

```diff
@@ -84,6 +180,14 @@ pub struct ReviewStatus {
     reviewer_completed_attestation: bool,
 }
 
+#[near(serializers=[borsh, json])]
+#[derive(Clone)]
+pub struct ReviewStatusV1 {
```

Comment:
I would call these new structs V2 and the old ones V1 to make it clear as currently it is not; which one is the latest?